crates/red_knot_test/src/lib.rs

```diff
@@ -221,18 +221,22 @@ fn run_test(
         }
     }
 
+    let configuration = test.configuration();
+
     let settings = ProgramSettings {
-        python_version: test.configuration().python_version().unwrap_or_default(),
-        python_platform: test.configuration().python_platform().unwrap_or_default(),
+        python_version: configuration.python_version().unwrap_or_default(),
+        python_platform: configuration.python_platform().unwrap_or_default(),
         search_paths: SearchPathSettings {
             src_roots: vec![src_path],
-            extra_paths: test
-                .configuration()
-                .extra_paths()
-                .unwrap_or_default()
-                .to_vec(),
+            extra_paths: configuration.extra_paths().unwrap_or_default().to_vec(),
             custom_typeshed: custom_typeshed_path.map(SystemPath::to_path_buf),
-            python_path: PythonPath::KnownSitePackages(vec![]),
+            python_path: PythonPath::KnownSitePackages(
+                configuration
+                    .python()
+                    .into_iter()
+                    .map(SystemPath::to_path_buf)
+                    .collect(),
+            ),
```

Comment:
Hmm, I'm still struggling to get my test to pass locally. I think this is incorrect because the `python` configuration setting is meant to point to the installation's `sys.prefix` value (e.g. `.venv`) but the `KnownSitePackages` variant of `PythonPath` is meant to hold the resolved `site-packages` subdirectory of `sys.prefix` (e.g. `.venv/lib/python3.13/site-packages` if it's a Python 3.13 virtual environment)

Reply:
I think this should be using `PythonPath::SysPrefix` instead

Reply:
I'll fix this as part of my PR

Reply:
🤦 yes, that should be `SysPrefix`, at least if it should mirror the CLI

Reply:
Yeah, I'm writing a fix!